snow/consensus/snowman/consensus_test.go

```diff
@@ -1319,6 +1194,12 @@ func LastAcceptedTest(t *testing.T, factory Factory) {
 	require.Equal(GenesisID, lastAcceptedID)
 	require.Equal(GenesisHeight, lastAcceptedHeight)
 
+	require.NoError(sm.RecordPoll(context.Background(), bag.Of(block0.IDV)))
+
+	lastAcceptedID, lastAcceptedHeight = sm.LastAccepted()
+	require.Equal(GenesisID, lastAcceptedID)
+	require.Equal(GenesisHeight, lastAcceptedHeight)
+
 	require.NoError(sm.RecordPoll(context.Background(), bag.Of(block1.IDV)))
 
 	lastAcceptedID, lastAcceptedHeight = sm.LastAccepted()
```

Comment:
block1 transitively votes for block0, and we receive 1 vote for block1 and 1 vote for block0. This means we accept block 0 and have a confidence counter of 1 for block 1 (needs one more poll to finalize).